server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -518,4 +740,22 @@ private void homerMovesTheMailInAnotherMailbox(String messageId) {
             .when()
             .post("/jmap");
     }
+
+    private void exportVaultContent(ExportRequest exportRequest) {
+        String taskId =
+            webAdminApi.with()
+                .body(exportRequest.matchingQuery)
+                .post("/deletedMessages/users/" + exportRequest.userExportFrom + "?action=export&exportTo=" + exportRequest.sharee)
+            .jsonPath()
+                .get("taskId");
+
+        webAdminApi.given()
+                .get("/tasks/" + taskId + "/await")
+            .then()
+                .body("status", is("completed"));
+    }
+
+    private ZipFile zipFile(InputStream data) throws IOException {
```

Comment:
This technical method should be abstracted away by ZipAssert related abstractions